roboRIO-2024/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -40,12 +41,15 @@
 import frc.robot.Constants.MotorConstants;
 import frc.robot.Constants.RatioConstants;
 import frc.robot.DashboardHelper.LogLevel;
+import frc.robot.util.BetterSlewRateLimiter;
 import frc.robot.DashboardHelper;
 
 public class Drivetrain extends SubsystemBase {
+  
   // There is a different system used than previous years because MotorControlGroup is deprecated :(.
   // We set a motor to a leader, and make followers follow the leader in the constructor.  
   // Front wheels are leaders for no reason because its redundant
+  BetterSlewRateLimiter slewRateLimiter = new BetterSlewRateLimiter(60, 0, 0);
```

Comment:
The deceleration limit should be a really high number, like 600, since right now you are saying the maximum deceleration is 0 units per second.